### DISCUSSION TITLE
Template Timer "custom:button-card"

### DISCUSSION THREAD

**Discussion by Marcoletto** [indomus/forum#1372] [HUB: Home Assistant]
Ciao a tutti,
sto impazzendo per inserire un timer con conto alla rovescia nell' angolino in alto di una "custom:button-card".

Ora, senza tediarvi con tutta la storia del codice e quello che ci sta dietro, io ho le seguenti cose da fare:

- inserire un bottone che cliccato mi comanda un "input_boolean.scaldabagno_temporizzato"
- inserire in alto a destra un timer che fa il conto alla rovescia "timer.scaldabagno"

Questo timer è attivato dall' input_boolean.scaldabagno_temporizzato e settato da una input_number

Ora, il problema sta nel fatto che ho fatto moltissime prove, anche aiutato da gente che ne sa ben più di me ma senza risultati
Il fatto è che se metto in custom_field il banale timer, vedo esclusivamente "IDLE" o "ACTIVE"; non vedo il conto alla rovescia.
Se metto un template che restituisce il "duration" quando clicco vedo il valore del timer ma poi non si aggiorna più, resta fermo al valore massimo.

Non saprei più dove sbattere la testa.

Ci sarebbe da creare un template ma non riesco a capire come.
Ne ho provato uno complicatissimo ma quando clicco il conto parte da "00 min" per poi continuare a salire e non fermarsi fino ad arrivare a "60 min" per poi ricominciare da "00 min".

Sapete aiutarmi?

Grazie

**Comment by Marcoletto** [comment]
Qualcuno mi sa aiutare?
Serve inserisca il codice di cosa ho attualmente implementato?

**Comment by Marcoletto** [comment]
Nessuna anima pia? :)

**Reply by 50494554524F** [reply]
prova ad usare mod-card e dentro ci metti quello che vuoi

**Comment by carlone1** [comment]
ciao ho fatto un automazione con un timer grafico, 
1- devi installare un frontend x il countdown grafico
2- clicco un pulsante e parte lo scaldabagno e il countdown
2- tempo 1 ora  e si spegne lo scaldabagno

ti metteri qui una immagine ma non so se si puo, non conosco il forum :)

**Reply by Marcoletto** [reply]
Ciao Carlone,
che frontend devo installare?
Mi sai indirizzare?

Grazie
